client/src/js/app.js

```diff
@@ -517,6 +517,23 @@ elmApp.ports.scrollToElement.subscribe(function(elementId) {
   waitForElement(elementId, scrollToElement, null);
 });
 
+elmApp.ports.getCoordinates.subscribe(function() {
+  if ("geolocation" in navigator) {
+    navigator.geolocation.getCurrentPosition(
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/getCurrentPosition#options

See the options here.
It would be meaningful to specify a timeout.
Also consider the others, `maximumAge` and `enableHighAccuracy`.